### PR TITLE
[Kiev-2019] add approved speakers

### PR DESCRIPTION
**Pull request by delgod**

### CODE REVIEW

File: content/events/2019-kiev/program/dan-barker.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
...is.  I have been studying this field for years, and we’ll discuss some of the original research as well as some of the latest related to DevOps. Then we’ll discuss some of the methods I’ve used in transformations that have included culture. In fact, all transformations are underpinned by culture, and that is something I always focus on first. What will the audience get? The audience will understand what is Just Culture, where it came from, and how to implement it at their companies. I’ll also provide some additional resources for the audience to learn more after the session. Why me? I often find myself the leader of transformations, but I’m merely the representative for so many who brought about these transformations. I brought large changes to a large medical company as a junior software engineer. I transformed a multinational, fifteen thousand person company as a DevOps Engineer which included cloud migrations and culture changes. Then I moved to a 150-year-old non-profit where I brought cloud, data, and culture changes through a large transformation. Most recently, I started a transformation at a large multinational security software vendor with culture, cloud, and SaaS changes.
```
Comment (phrawzty) [inline content/events/2019-kiev/program/dan-barker.md]: This starts with "Level Beginner Short". Might want to remove that?

File: content/events/2019-kiev/program/toshaan-bharvani.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
++++
+Talk_date = ""
+Talk_start_time = ""
+Talk_end_time = ""
+Title = "Comparing DevOps to pasta"
+Type = "talk"
+Speakers = ["toshaan-bharvani"]
++++
+
+The presentation describes how IT infrastructure has evolved from spaghetti infrastructure, a setup where organic growth and individual parts of the system were inter weaved into the architecture where developers were operations, the prevalence of virtualization and the increase in IT usage created individual layers in the infrastructure and created silos of specialization and this created a difference between developers making software and operations managing the infrastructure. From this lasagna based system, that was cumbersome and difficult to manage the idea of developer techniques and methodology were added to create teams, describing infrastructure as code. This lead to team being able to deliver software faster and deploy easier, the infrastructure was still not as agile, however in an effort to make infrastructure to be as agile the idea of containers where used to make infrastructure more agile, creating small environments for specific usage, to be able to manage it easier and quicker, and so
```
Comment (phrawzty) [inline content/events/2019-kiev/program/toshaan-bharvani.md]: This ends with "and so", then nothing. Missing more words?

File: content/events/2019-kiev/speakers/julia-tkachova.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
++++
+Title = "Julia Tkachova"
+Twitter = ""
+image = "julia-tkachova.png"
+type = "speaker"
+linktitle = "julia-tkachova"
+
++++
+
+Blonde in tech with 8 years in industry, was part of devops adoption in few companies, building processes and teams, changing world on a daily basis.
```
Comment (phrawzty) [inline content/events/2019-kiev/speakers/julia-tkachova.md]: "Blonde in tech" is borderline with regardless to the CoC. Consider rephrasing.

File: content/events/2019-kiev/speakers/ruslan-kusov.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+image = "ruslan-kusov.png"
+type = "speaker"
+linktitle = "ruslan-kusov"
+
++++
+
+9 years of experience in IT industry as a System Administrator, Network Engineer, DevOps and DevOps Architect.
+Created and optimized architecture of different projects for World-known tourism, payment services and mobile development companies.
+Have extensive background in Google (GCP) and Amazon (AWS) cloud platforms, Java/Web application platforms, IAC and provisioning tools, NoSQL/SQL Databases, On-Premises Virtualization, Kubernetes, CI/CD systems.
+Certified Google Cloud Professional Architect.
```
Comment (phrawzty) [inline content/events/2019-kiev/speakers/ruslan-kusov.md]: This won't render as a bullet list. You need more line-breaks, basically. 🙂 

image

File: content/events/2019-kiev/program/dan-barker.md (hunk 1 above)
Comment (delgod) [reply content/events/2019-kiev/program/dan-barker.md]: fixed, thank you

File: content/events/2019-kiev/speakers/julia-tkachova.md (hunk 3 above)
Comment (delgod) [reply content/events/2019-kiev/speakers/julia-tkachova.md]: "Blonde in tech" phrase was provided by @jtkachova personaly, asking for rephrasing.

File: content/events/2019-kiev/speakers/ruslan-kusov.md (hunk 4 above)
Comment (delgod) [reply content/events/2019-kiev/speakers/ruslan-kusov.md]: fixed, thank you!

File: content/events/2019-kiev/program/toshaan-bharvani.md (hunk 2 above)
Comment (delgod) [reply content/events/2019-kiev/program/toshaan-bharvani.md]: fixed, thank you!

File: content/events/2019-kiev/speakers/julia-tkachova.md (hunk 3 above)
Comment (julytc) [reply content/events/2019-kiev/speakers/julia-tkachova.md]: It's a pity that I can't use "blonde in tech", I believe there is nothing bad in self-ironic, however, let's rephrase it to "lady in tech". Thank you.
